lib/mk-hashi-stack.nix

```diff
@@ -1,9 +1,12 @@
-{ rootDir, pkgs, flake, domain # E.g. "mantis.ws"
+{ flake, domain # E.g. "mantis.ws"
 , dockerRegistry ? "docker." + domain, dockerRole ? "developer"
 , vaultDockerPasswordKey ? "kv/nomad-cluster/docker-developer-password" }:
 
 let
-  lib = pkgs.lib;
+  inherit (flake.inputs) nixpkgs;
+  inherit (nixpkgs) lib;
+
+  pkgs = nixpkgs.legacyPackages.x86_64-linux.extend flake.overlay;
```

Comment:
the `rootDir` -> `flake` and overlay change are breaking changes. Not to mention that only applying one overly is probably not going to be sufficient for some other repos. 4 are used for one of our repositories.

Reply:
https://github.com/kreisys/flake-utils/blob/master/simpleFlake.nix#L70